.github/workflows/rust.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         - os: ubuntu-latest
           headless: Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
         - os: ubuntu-latest
-          dependencies: sudo apt install libxtst-dev
+          dependencies: sudo apt-get install libxtst-dev
```

Comment:
I would rather we keep `apt` instead of  `apt-get`.
(https://itsfoss.com/apt-vs-apt-get-difference/)

Reply:
apt-get maintains backward compatibility, whereas apt does not - it is designed for humans [source](https://askubuntu.com/questions/445384/what-is-the-difference-between-apt-and-apt-get#446484). Using apt-get in scripts is a best practice. It should actually be `sudo apt-get install libxtst-dev --assume-yes`

Reply:
Fair enough ! I learned something.